src/_scss/pages/homepage/homepage.scss

```diff
@@ -3,6 +3,19 @@
   @import "layouts/default/default";
   @import "layouts/default/header/_warning";
 
+  .flex-gap {
+    display: inline-flex;
+    flex-wrap: wrap;
+  }
+
+  .flex-gap > div {
+    width: 20px;
+    height: 20px;
+    margin: 6px;
+    background-color: lightblue;
```

Comment:
what is that background color doing?

Reply:
removed, though this was for the tempPage with the card usage examples